### PR TITLE
fix intg timing bug

### PR DESCRIPTION
**Pull request by QuiteStochastic**
fixing the bug in accordance with deepthi's suggestions

### CODE REVIEW

File: src/lib/integration_test_cloud_engine/coda_automation.ml [hunk 1]
```diff
@@ -109,7 +109,10 @@ module Network_config = struct
     (* GENERATE ACCOUNTS AND KEYPAIRS *)
     let num_block_producers = List.length block_producers in
     let block_producer_keypairs, runtime_accounts =
-      let keypairs = Array.to_list (Lazy.force Sample_keypairs.keypairs) in
+      (* the first keypair is the genesis winner and is assumed to be untimed. Therefore dropping it, and not assigning it to any block producer *)
+      let keypairs =
+        List.drop (Array.to_list (Lazy.force Sample_keypairs.keypairs)) 1
```
Comment (psteckler) [inline src/lib/integration_test_cloud_engine/coda_automation.ml:114]: would `List.tl_exn` be simpler?

File: src/lib/integration_test_cloud_engine/coda_automation.ml [hunk 2]
```diff
@@ -109,7 +109,10 @@ module Network_config = struct
...
+      in
       if List.length block_producers > List.length keypairs then
```
Comment (psteckler) [inline src/lib/integration_test_cloud_engine/coda_automation.ml:116]: can use `num_block_producers` here, rather than recalculating length

Comment (QuiteStochastic) [reply src/lib/integration_test_cloud_engine/coda_automation.ml:116]: what do you mean?

**Review by nholland94 — APPROVED** [review]

File: src/lib/integration_test_cloud_engine/coda_automation.ml (hunk 2 above)
Comment (QuiteStochastic) [reply src/lib/integration_test_cloud_engine/coda_automation.ml:116]: oh i see what you mean, sure

**Pull request merged**